rest_framework/routers.py

```diff
@@ -16,9 +16,8 @@
 import itertools
 from collections import OrderedDict, namedtuple
 
-from django.conf.urls import url
 from django.core.exceptions import ImproperlyConfigured
-from django.urls import NoReverseMatch
+from django.urls import NoReverseMatch, path, re_path
```

Comment:
So, is path used? (What did `flake8` say?)

Reply:
Hi @carltongibson ,
flake8 has passed the code successfully.